### PR TITLE
add admis app to allow superuser to trigger tasks to rebuild the cache

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: ureport/admins/views.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+class AdminCRUDL(SmartCRUDL):
+    actions = ('refresh_cache',)
+    model = Org
+
+    class RefreshCache(SmartUpdateView):
+        fields = ('id',)
+        success_message = None
+        success_url = '@orgs.org_home'
+
+        def pre_process(self, request, *args, **kwargs):
```
Comment (nicpottier) [inline ureport/admins/views.py]: This shouldn't be in pre_process, everything in here is done before permissions are checked etc.. instead it should be in post_save() or the like.

File: ureport/admins/tasks.py [hunk 2]
```diff
@@ -0,0 +1,160 @@
+import time
```
Comment (nicpottier) [inline ureport/admins/tasks.py]: Why did you move all these tasks over from /polls to /admins? 

Seems like the polls app was the more logical place for them.

Comment (norkans7) [inline ureport/admins/tasks.py]: wanted these to be used both by the beat scheduler and the refresh cache view

I can replace them back

File: ureport/admins/views.py (hunk 1 above)
Comment (norkans7) [inline ureport/admins/views.py]: ok

File: ureport/admins/tasks.py (hunk 2 above)
Comment (nicpottier) [inline ureport/admins/tasks.py]: Ya, to me it feels like tasks that update polls do belong in the /polls app, while the admins app can certainly fire off one of those tasks, it isn't really in charge of polls, the polls app is.

**Comment by norkans7** [comment]
Please review, I think this is ready

File: ureport/admins/__init__.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    main_polls = 2
+    brick_polls = 3
+    other_polls = 4
+    flows = 5
+    reporters = 6
+    graphs_data = 7
+
+
+def refresh_caches(org, caches):
+    if not org:
```
Comment (nicpottier) [inline ureport/admins/__init__.py]: Why would org ever be None?

**Comment by nicpottier** [comment]
Looks good.

File: ureport/admins/__init__.py (hunk 3 above)
Comment (norkans7) [inline ureport/admins/__init__.py]: right, I remove this, this is unexpected
